data/root/etc/inst_setup

```diff
@@ -55,7 +55,7 @@ unset SSH_FAILED
 stty sane 2>/dev/null
 
 # get hostname & hostips
-hostip_from_wicked >/tmp/host_ips 2>/tmp/host_name
+hostip_from_wicked /tmp/host_ips /tmp/host_name
```

Comment:
> Common practice is to have one method to do just one thing

I'd follow you for ruby methods or oo in general, but not shell scripts. If the name irks you, we can rename
the script to something else.

> So at least documentation should be fixed.

It does exactly what is documented.